### PR TITLE
Added jest & babel to run typescrypt tests

### PR DESCRIPTION
**Pull request by vkuts**

### CODE REVIEW

File: packages/daemon/src/veramo/__tests__/vc-management.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      veramoSecret: '29739248cad1bd1a0fc4d9b75cd4d2990de535baf5caadfdf8d8f86664aa830c',
+      infuraProjectId: '9386bd3e92034b6784bc17ed1d7253f4',
+    }));
+  })
+  afterAll(_clearVeramo);
+
+  it('should create Daemon service did on start', async () => {
+    const daemonDid = await findDaemonDid();
+    expect(daemonDid).not.toBeNull();
+  })
```
Comment (vkuts) [inline packages/daemon/src/veramo/__tests__/vc-management.ts:19]: Now able to write more of these kinds of tests exercising the backend lib and verifying that it does the right thing

File: packages/daemon/src/veramo/did-manager.ts [hunk 2]
```diff
@@ -1,16 +1,45 @@
...
 
+// requires calling initVeramo to create this local singleton agent and dbConnection
+let agent: TAgent<IDIDManager & IKeyManager & IDataStore & IDataStoreORM & IResolver>;
+let dbConnection: Connection;
+let dbFile: string;
 /**
  * Truing up the local Veramo DID store.
  */
-export const initDids = async () =>
+export const initVeramo = async (overrides: VeramoAgentConfigOverrides = null) => {
```
Comment (vkuts) [inline packages/daemon/src/veramo/did-manager.ts:15]: added ability to override the setup to separate test DB from regular one

File: packages/daemon/src/veramo/did-manager.ts [hunk 3]
```diff
@@ -1,16 +1,45 @@
...
 
+  return agent;
+};
+
+/**
+ * A utility for integration tests. Don't use in production code!
+ */
+export const _clearVeramo = async () => {
+  try {
+    await (await dbConnection).dropDatabase();
```
Comment (nathan-knight) [inline packages/daemon/src/veramo/did-manager.ts]: Why do you have to nest `await dbConnection` twice? Can't you await it once and store the result?

Comment (vkuts) [reply packages/daemon/src/veramo/did-manager.ts]: Good point. It was copy/paste code from their repo and they stored a promise for the connection for some reason. It's redundant here.

**Pull request merged**